src/main/java/com/docusign/esign/client/ApiClient.java

```diff
@@ -766,7 +766,13 @@ private ClientResponse getAPIResponse(String path, String method, List<Pair> que
     ClientResponse response = getAPIResponse(path, method, queryParams, body, headerParams, formParams, accept, contentType, authNames);
 
     if (response.getStatusInfo().getFamily() != Family.SUCCESSFUL) {
-        throw new ApiException("Error while requesting server, received HTTP code: " + response.getStatusInfo().getStatusCode() + " / with response Body: " + response.getEntity(String.class));
+        String respBody = null;
```

Comment:
@psytester can you change it to: String respBody = response.getEntity(String.class);